modules/mcuboot/CMakeLists.txt

```diff
@@ -314,6 +322,12 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
 
     set(app_offset $<TARGET_PROPERTY:partition_manager,app_TO_SECONDARY>)
 
+    if (CONFIG_NRF53_ENFORCE_IMAGE_VERSION_EQUALITY)
+      set(sign_dependencies IMAGE_DEPENDENCIES "(1,${CONFIG_MCUBOOT_IMAGE_VERSION})")
```

Comment:
here 1 is a magic value to me, should we pull that into a variable or prompt-less kconfig so its clearer what it means?

Reply:
I see that both `0` and `1` are hard-coded as magic values in many places here within this file. I will try to make it better then.

Reply:
After a call with @nvlsianpu we decided to pass it through CMake variables (there is no need to know this variable in other contexts). @hakonfam Do you find it acceptable?

Additionally - I have verified that magic numbers inside the multi-image builder should not use those symbols as they have a separate ID number space.

One remaining place could be the magic parameters that generate ZIP upgrades, but I do not have knowledge on that one:
```
      set(generate_script_params
        "${app_core_binary_name}load_address=$<TARGET_PROPERTY:partition_manager,PM_APP_ADDRESS>"
        "${app_core_binary_name}image_index=0"
        "${app_core_binary_name}slot_index_primary=1"
        "${app_core_binary_name}slot_index_secondary=2"
        "${app_core_binary_name}version_MCUBOOT=${CONFIG_MCUBOOT_IMAGE_VERSION}"
        "${net_core_binary_name}image_index=1"
        "${net_core_binary_name}slot_index_primary=3"
        "${net_core_binary_name}slot_index_secondary=4"
        "${net_core_binary_name}load_address=$<TARGET_PROPERTY:partition_manager,CPUNET_PM_APP_ADDRESS>"
        "${net_core_binary_name}board=${CONFIG_DOMAIN_CPUNET_BOARD}"
        "${net_core_binary_name}version=${net_core_version}"
        "${net_core_binary_name}soc=${net_core_soc}"
        )
```
Since they are using values greater than 1, I assumed that they also have their own ID/index space.